### PR TITLE
Update gitops.md

### PR DESCRIPTION
**Pull request by akshatcoder-hash**

### CODE REVIEW

**Comment by akshatcoder-hash** [comment]
Hey @CathPag , please have a look here
image

File: content/en/gitops.md [hunk 1]
```diff
@@ -4,17 +4,35 @@ status: Feedback Appreciated
 category: Concept
 ---
 
+
 ## What it is
+Let's start with DevOps and then move on to GitOps. 
+**DevOps** is a set of methods that automate and integrate processes between development and operations teams.
```
Comment (CathPag) [inline content/en/gitops.md:10]: There is no need to define DevOps as we already have a definition.

File: content/en/gitops.md [hunk 2]
```diff
@@ -4,17 +4,35 @@ status: Feedback Appreciated
...
+
+It integrates operations such as Git deployment, management, and monitoring for containerized clusters and softwares.
+
+The Git repository for the project stores the entire state of the system, making the history of system changes visible and editable.
 
-GitOps is a set of best practices based on [shared principles](https://opengitops.dev/), applied to a workflow that depends on software agents that enable automation to reconcile a declared system state or configuration in a git repository.
-These software agents and practices are used to execute a cohesive workflow that leverages a source control system like Git as the “single source of truth” and extends this practice to applications, infrastructure, and operational procedures.
+These software agents and practices are used to execute a cohesive workflow that leverages a source control system like Git as the "single source of truth" and extends this practice to applications, infrastructure, and operational procedures.
```
Comment (CathPag) [inline content/en/gitops.md:19]: Same here. Let's not talk about agents. It requires explaining what it is and introduces a lot of unnecessary complexity. What is a cohesive workflow that leverages source control systems?

File: content/en/gitops.md [hunk 3]
```diff
@@ -4,17 +4,35 @@ status: Feedback Appreciated
...
 
-## Problem it addresses
 
-Existing processes for infrastructure configuration management can face challenges such as configuration drift, failed deployments, relying on a system's previous state for success, missing documentation, or unknown development history.
-Adopting a GitOps workflow can help alleviate these issues, among several others.
+
+## Problems it addresses
```
Comment (CathPag) [inline content/en/gitops.md:23]: Problem it addresses should not talk about GitOps at all and should focus on the problem only. What is life without GitOps like?

File: content/en/gitops.md [hunk 4]
```diff
@@ -4,17 +4,35 @@ status: Feedback Appreciated
...
+- Git eliminates the need of relying on previous software state for successful deployment.
```
Comment (CathPag) [inline content/en/gitops.md:24]: Please make sure to keep the style consistent. We don't use bullet points in the glossary

**Review by CathPag — COMMENTED** [review]
Hi there, thanks for your contribution. This needs some additional work. Please see my comments.

**Comment by akshatcoder-hash** [comment]
Hey Cathy.
Thanks for your feedback. I'll be working on it 👍

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by akshatcoder-hash** [comment]
Hey @CathPag , I need some more time to work on this.

**Comment by CathPag** [comment]
> Hey @CathPag , I need some more time to work on this.

I'm glad to hear. This is a difficult term and can't be done quickly :) We had a team of GitOps pros working on it and we didn't get to a point where it was mergeable. Feel free to pick an easier term too, if this turns out to be too tough.

**Comment by CathPag** [comment]
Hi @akshatcoder-hash, are you still interested in working on this? If not, I'll go ahead and close this PR and free up the issue.

**Comment by akshatcoder-hash** [comment]
Hey @CathPag , you can close the PR

**Comment by akshatcoder-hash** [comment]
Hey @CathPag , you can go on closing it

**Comment by CathPag** [comment]
Ok, thank you!

**Pull request closed without merging**